### PR TITLE
Added a temporary fix for the pause container in the dashboard

### PR DESCRIPTION
**Pull request by mssola**
This fix should eventually go away once we refactor how we provision the
dashboard node.

Signed-off-by: Miquel Sabaté Solà <msabate@suse.com>

### CODE REVIEW

File: activate.sh [hunk 1]
```diff
@@ -22,6 +22,9 @@ fi
 cp -v $manifest_dir/salt.yaml $kube_dir
 cp -v $manifest_dir/velum.yaml $kube_dir
 
+# Make sure that the controller node looks for the local pause image
```
Comment (inercia) [inline activate.sh]: A `TODO` would remind us to remove this aberration... :-)

Comment (mssola) [inline activate.sh]: Done

Comment (jordimassaguerpla) [inline activate.sh]: +1

Comment (jordimassaguerpla) [inline activate.sh]: lgtm because it is the minimal fix for this ...

**Pull request merged**